buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt

```diff
@@ -26,6 +26,7 @@ private const val COROUTINES_VERSION = "1.4.1"
 private const val DEXTER_VERSION = "6.2.1"
 private const val DRAWABLETOOLBOX_VERSION = "1.0.7"
 private const val DOTENV_VERSION = "5.1.4"
+private const val DOKKA_VERSION = "1.4.10.2"
```

Comment:
Hate to be that guy all the time, but the new lines in this file are not sorted correctly alphabetically 😅

Reply:
I am the one that created this file and started to use this order and I needed to double-check it because I was pretty sure it was sorted correctly. 
Good catch!!!